File: task03/python/subtask4.py
with open("input.txt", "r") as f:
    n = int(f.read())

with open("output.txt", "w") as f:
    for i in range(n):
        f.write(" " * (n - i - 1) + "*" * (2 * i + 1) + "\n")

    for i in range(n - 2, -1, -1):
        f.write(" " * (n - i - 1) + "*" * (2 * i + 1) + "\n")

File: task03/elixir/subtask4.exs
n = File.read!("input.txt") |> String.trim |> String.to_integer

File.write!("output.txt", "")

for i <- 0..n-1 do
  IO.write(:stdio, " " <> String.duplicate(" ", n - i - 1) <> String.duplicate("*", 2 * i + 1) <> "\n")
end

for i <- n-2..0 do
  IO.write(:stdio, " " <> String.duplicate(" ", n - i - 1) <> String.duplicate("*", 2 * i + 1) <> "\n")
end
